ui_framework/components/button/plus_button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { KuiButton } from './kui_button';
+import { PlusIcon } from '../icon/plus_icon';
```

Comment:
Sorry to make this comment late, but I think we should name these semantically, e.g. PlusIcon -> CreateIcon, PlusButton -> CreateButton, TrashButton -> DeleteButton. This is because we may want to change the icons that represent these actions, and it will be easier to change them within the implementation and have the entire UI update, than to change each use case one-by-one.